qt/settingsdialog.py

```diff
@@ -1009,14 +1009,14 @@ def __init__(self, parent):
         # additional rsync options
         hlayout = QHBoxLayout()
         layout.addLayout(hlayout)
+        tooltip = _('Options must be quoted e.g. {example}.').format(
```

Comment:
Is quotation required for the option name or option value or both?
The example looks like quotation is required for the option value only.
In this case I would clarify this in the tooltip, eg. via

'Option values must be quoted, e.g. {example}'

Reply:
No that is OK that way. The result in GUI (Settings -> Export Options, with SSH profile).

![grafik](https://github.com/bit-team/backintime/assets/11861496/e951f339-d012-4a2e-87e0-6d3b26a1bdd9)